model/src/main/proto/interaction_object.proto

```diff
@@ -30,6 +31,11 @@ message StringList {
   repeated string html = 1;
 }
 
+// Structure containing a ratio object.
+message RatioExpression {
+  repeated uint32 ratio_component = 1;
```

Comment:
Suggest mentioning that it's expected that this list be more 1 element. 0 and 1 elements are invalid states.

Reply:
Updated

Reply:
Dond